.setup/apache/submitty.conf

```diff
@@ -1,144 +0,0 @@
-# If hosting more than just Submitty on Apache, you should set the
```

Comment:
oops, I don't think your PR should be deleting this file.  That is likely why its failing the travis e2e tests

Reply:
Readded this file.